hub/core/transform/test_transform.py

```diff
@@ -0,0 +1,96 @@
+from hub.api.dataset import Dataset
+from hub.core.transform.transform import transform
+import numpy as np
+
+
+def fn1(i, mul=1, copy=1):
+    d = {}
+    d["image"] = np.ones((337, 200)) * i * mul
+    d["label"] = np.ones((1,)) * i * mul
+    return [d for _ in range(copy)]
+
+
+def fn2(sample, mul=1, copy=1):
+    d = {}
+    d["image"] = sample["image"] * mul
+    d["label"] = sample["label"] * mul
+    return [d for _ in range(copy)]
+
+
+def fn3(i, mul=1, copy=1):
+    d = {}
+    d["image"] = np.ones((4310, 2187)) * i * mul
+    d["label"] = np.ones((13,)) * i * mul
+    return [d for _ in range(copy)]
+
+
+def test_transform_hub_dataset(local_storage):
+    with Dataset("./test/transform_hub_in") as data_in:
```

Comment:
```suggestion
@parametrize_all_dataset_storages_and_caches
def test_transform_hub_dataset(ds):
    with ds as data_in:
```

please use the ds fixture everywhere you can.

if you need `storage` you can do `ds.storage`

uhhhhh..... actually is this even possible for the `with` statement?????

Reply:
maybe


```python
with ds:
    pass
```